doc/manuals/admin/database_model.md

```diff
@@ -325,6 +325,8 @@ Fields:
 -   **lastSuccessCode**: HTTP code (200, 400, 404, 500, etc.) returned by receiving endpoint last
     time a successful notification was sent.
     Not present if the subscription has never provoked a successful notification.
+-   **maxFailsLimit**: An optional field used to specify the maximum limit of connection attempts, so when that number of failing notifications is reached, then the subscription passes automatically to inactive state.
+-   **failsCounter**: the number of failing notifications associated to the subscription.
```

Comment:
Regardless any other document describing the functionality in detail, I think is a good idea to explain how the failCounter field is modified. Thus, I'd suggest the following modification:

```suggestion
-   **failsCounter**: the number of failing notifications associated to the subscription. This is increased by one each time a notification attempt fails. It is reset to 0 if a notification attempt successes or the subscription transition from "inactive" to "active" status.
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/6b68a8c9ded45dd2e30c033f4d5000cddfab19de